src/adapters/github.ts

```diff
@@ -309,7 +309,10 @@ class GithubAdapter extends GitAdapter {
   }
 
   protected getRepositoryUrl(repo: IRepo): string {
-    return `git@${shepherdGitHubEnterpriseUrl}:${repo.owner}/${repo.name}.git`;
+    return (
+        `git@${shepherdGitHubEnterpriseUrl}:${repo.owner}/${repo.name}.git`
```

Comment:
Drop `.replace` call.

Reply:
Rename to gitHubEnterpriseUrl

Reply:
This needs to be base url, not full url (e.g., api.github.com vs. api.github.com/api/v3)